app/assets/javascripts/get_neighborhoods.js

```diff
@@ -36,13 +39,17 @@ function getNeighborhoods (address1, address2, address3, trans_type) {
 }
 
 function appendNeighborhoods(data) {
-  for (var i = 0; i < data.length; i++) {
-    $('#return-addresses').append(createNeighborhoodHTML(data[i]));
+  for (var i = 0; i < data[0].length; i++) {
+    $('#return-addresses').append(createNeighborhoodHTML(data[0][i]));
+  }
+  for (var i = 0; i < data[1].length; i++) {
+    $('#return-rent').append(createNeighborhoodRentHTML(data[1][i]));
```

Comment:
I think this is a good temporary solution to have both neighborhood priorities listed on the page.